src/touchhandler/TouchHandler.h

```diff
@@ -6,13 +6,25 @@ namespace Pinetime {
   namespace Controllers {
     class TouchHandler {
     public:
+      // Enum describes how the watch was woken:
```

Comment:
I'm wondering if this new functionality allowing to ignore touch events should be in the TouchHandler, or if it should be done at the SystemTask level (which wouldn't call the TouchHandler if we decide to ignore the TouchEvent) or in another object that would only do that.

This would allow to maintain the "single responsibility principle" on TouchHandler.

Or perhaps there were other reasons to implement this management in TouchHandler?

Reply:
moved responsibility from `TouchHandler` to `SystemTask`. And simplified a bit